src/routes/[id]/training/+page.svelte

```diff
@@ -44,12 +44,16 @@
       <Card.Header>
         <Card.Title>Training Queues</Card.Title>
       </Card.Header>
-      <Card.Content class="space-y-1.5">
+      <Card.Content class="space-y-2">
         {#if data.memberOfQueue !== null}
           <p>
             Enrolled in the <b>{data.memberOfQueue.name}</b>
             queue.
           </p>
+          <p>
+            You are number <b>{data.position}</b>
```

Comment:
Does this just show -1 if the backend query fails? Perhaps instead we redirect the user with a toast or a dialog complaining.

Reply:
fc00532f6141728f131025ec10ca15f5e069e8a5

I think this sorts it?